index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>DSTEAM</title>
+
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+		integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+	<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.min.js"></script>
+	<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
+
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link
+		href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,400;1,700;1,900&display=swap"
+		rel="stylesheet">
+	<link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+
+	<link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+
+	<header>
+		<nav class="display-none fixed-top" id="nav-destop">
+			<div class="topBar">
```

Comment:
- [optional] please kindly make your mobile menu full width and responsive.you can also add the other menu links to it (program, join, sponsors etc) so they can facilitate movement to different page sections.  nice work
![Screenshot (993)](https://user-images.githubusercontent.com/65298048/233734410-78c41f8c-56fd-4d69-930a-68d21d772170.png)

Reply:
Ok, thanks for the recommendations. 👍🏼